addons/mass_mailing_sms/tests/test_mailing_sms_ui.py

```diff
@@ -0,0 +1,43 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo.tests
```

Comment:
```suggestion
from odoo.tests import tagged, users
```
-> users for the @users decorator proposal